src/components/DurationMenu.js

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+
+function DurationMenu (props) {
+
+    // For making a visable focus since the radio is hidden
+    const [focusOn, setFocusOn] = useState('');
+    const onFocus = (e) => {
+        setFocusOn(`${e.target.value}`)
+    }
+
+    // Remove focus styling from radios when focus moves to button
+    const onBlur = () => {
+        setFocusOn('');
+    }
+
+    //Send selected value up to calling component
+    const onClick = (e) => {
+        props.onSelect(e.target.attributes.for.value);
+    }
+
+    const durationToTimeSpan = (duration) => {
+        return duration === 'Annual' ? 'year' : 'month';
+    }
+
+    return (
+        <div className='flex flex-col items-center xxs:flex-row m-4'>
+            {props.options.map(option => {
+                return (
+                    <div key={option.duration} className={`inline m-1 p-4 pb-0 w-full xxs:w-1/2 bg-very-pale-blue rounded-xl ${focusOn === option.duration && 'outline outline-bright-blue'}`}>
+
+                        <label htmlFor={option.duration}
+                        onClick={onClick}
```

Comment:
We might need to have this `onClick` function to the `input` element rather than `label`. Reason being if someone is using a keyboard to navigate through this form, when they tab through the elements they will land on the input. And then if they use the enter key to select the input, I'm not sure the current label `onClick` will get detected

Reply:
Yes, I think I forgot to think that through properly. It probably is meant to be on the input.